consensus/src/processes/pruning_proof/mod.rs

```diff
@@ -723,6 +1093,87 @@ impl PruningProofManager {
             .collect_vec()
     }
 
+    /// BFS forward iterates from genesis_hash until selected tip, ignoring blocks in the antipast of selected_tip.
+    /// For each block along the way, insert that hash into the ghostdag_store
+    /// If we have a required_block to find, this will return true if that block was found along the way
+    fn fill_level_proof_ghostdag_data(
+        &self,
+        genesis_hash: Hash,
```

Comment:
The name `genesis_hash` is a bit confusing, bcs we usually only use it for the real genesis. Should this be simply `root` ?

Reply:
It is. Will change it to just `root`

Reply:
Fixed in [5cd27f3](https://github.com/kaspanet/rusty-kaspa/pull/494/commits/5cd27f3b71d60f0f9b646a67d75a5a3d50d6b2c2)